src/main/java/Binks.java

```diff
@@ -0,0 +1,81 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Binks {
+    /**
+     * Prints out the greeting message from the chatbot "Binks".
+     */
+    public static void greetUser(){
```

Comment:
Relevant statements serving the same function are grouped neatly under a function. This contributes to a highly logical coding structure.